docs/interpreter/influxdb.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: page
+title: "InfluxDB Interpreter for Apache Zeppelin"
+description: "InfluxDB is an open-source time series database designed to handle high write and query loads."
+group: interpreter
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# InfluxDB Interpreter for Apache Zeppelin
+
+<div id="toc"></div>
+
+## Overview
+[InfluxDB](https://v2.docs.influxdata.com/v2.0/)  is an open-source time series database (TSDB) developed by InfluxData. It is written in Go and optimized for fast, high-availability storage and retrieval of time series data in fields such as operations monitoring, application metrics, Internet of Things sensor data, and real-time analytics.
+This interpreter allows to perform queries in [Flux Language](https://v2.docs.influxdata.com/v2.0/reference/flux/) in Zeppelin Notebook.
+
+### Notes
+* This interpreter is compatible with InfluxDB 2.0+ (v2 API, Flux language) 
```

Comment:
According to README, the client library is compatible with 1.8 as well: https://github.com/influxdata/influxdb-client-java#influxdb-18-api-compatibility

Reply:
Thanks @alexott for the review. I will update the dependency to the latest InfluxDB client library and update README.